dev/ese/src/ese/bf.cxx

```diff
@@ -35,7 +35,7 @@ ERR ErrBFInit( __in const LONG cbPageSizeMax )
     g_dblBFHashLoadFactor     = 5.0;
     g_dblBFHashUniformity     = 1.0;
     g_csecBFLRUKUncertainty   = 1.0;
-
+ 
```

Comment:
You've done something to change random white spaces in this file.  Figure that out and undo it.  Happened in a few other files.   Or did this happen back in our internal branch!?  :-(

Reply:
I honestly don't know. I also thought that I fixed it, but I must have only done it in the mirror branch

Reply:
Or the other PR?